libbeat/logp/metrics.go

```diff
@@ -71,6 +71,9 @@ var gauges = map[string]bool{
 	"beat.memstats.memory_total":     true,
 	"beat.memstats.memory_alloc":     true,
 	"beat.memstats.gc_next":          true,
+	"beat.cpu.usage":                 true,
```

Comment:
@tsg At one point we should review these metrics to align them with `metricbeat` so it would be `system.cpu.*` .